openshift/templates/dancer-mysql.json

```diff
@@ -329,87 +329,88 @@
   "parameters": [
     {
       "name": "MEMORY_LIMIT",
-      "displayName": "Memory limit",
+      "displayName": "Memory Limit",
       "description": "Maximum amount of memory the Perl Dancer container can use",
       "value": "512Mi"
     },
     {
       "name": "MEMORY_MYSQL_LIMIT",
-      "displayName": "Memory limit",
+      "displayName": "Memory Limit (MySQL)",
       "description": "Maximum amount of memory the MySQL container can use",
       "value": "512Mi"
     },
     {
       "name": "SOURCE_REPOSITORY_URL",
-      "displayName": "Source repository URL",
+      "displayName": "Source Repository URL",
```

Comment:
There is talk of supporting other SCMs over time ... but sure, let's be consistent (and more precise), and worry about making it generic when it becomes generic.